src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/CSharpInstructionDecoder.cs

```diff
@@ -36,7 +36,8 @@ private CSharpInstructionDecoder()
 
         internal override string GetCompactName(MethodSymbol method)
         {
-            return (method.AssociatedSymbol ?? method).Name;
+            var symbol = method.AssociatedSymbol ?? method;
+            return symbol.ToDisplayString(CompactNameFormat);
```

Comment:
Why did you change this from `Name` to DisplayString?

Reply:
We're using `ToDisplayString()` so the member name in the string returned from `IDkmLanguageInstructionDecoder.GetMethodName()` is consistent regardless of whether `DkmVariableInfoFlags.CompactName` is used. See table in PR description.